sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/implementation/utils/TestUtils.java

```diff
@@ -27,8 +27,12 @@ public final class TestUtils {
             + "IngestionEndpoint=https://test.in.applicationinsights.azure.com/;"
             + "LiveEndpoint=https://test.livediagnostics.monitor.azure.com/";
 
+    public static TelemetryItem createMetricTelemetry(String name, int value, String connectionString) {
+        return createMetricTelemetry(name, value, connectionString, null, null);
```

Comment:
then you don't need the if/else below
```suggestion
        return createMetricTelemetry(name, value, connectionString, "state", "blocked");
```